fv_be/app/models/dictionary.py

```diff
@@ -0,0 +1,154 @@
+from django.db import models
+from django.utils.translation import gettext as _
+
+# FirstVoices
+from .sites import BaseSiteContentModel, BaseControlledSiteContentModel
+from .category import Category
+from .part_of_speech import PartOfSpeech
+
+
+class Note(BaseSiteContentModel):
+    """Model for notes associated to each dictionary entry."""
+
+    # from fv:notes,fv:general_note, fv:cultural_note, fv:literal_translation, fv-word:notes, fv-phrase:notes
+    text = models.TextField()
+    dictionary_entry = models.ForeignKey(
+        "DictionaryEntry", on_delete=models.CASCADE, related_name="notes"
+    )
+
+    def __str__(self):
+        return self.text
+
+
+class Acknowledgement(BaseSiteContentModel):
+    """Model for acknowledgments associated to each dictionary entry."""
+
+    # from fv:acknowledgments, fv:source, fv:reference, fv-word:acknowledgement, fv-phrase:acknowledgement
+    text = models.TextField()
```

Comment:
Similar to the Notes-- does this need to be a TextField or could it be a CharField with a maximum length?

Reply:
Max length can be added after research on basis of ticket FW-4196. Marking it resolved for now.